css/main.css

```diff
@@ -0,0 +1,991 @@
+    no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.fourthSectionBoxLatestRight5 {
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/07/Untitled-design80.png")
+    no-repeat;
+  background-size: cover;
+  background-position: 50%;
+  width: 30%;
+}
+
+.fourthSectionBoxImg6 {
+  margin-top: 15px;
+  height: 276px;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/07/cryptocurrency-q2-second-quarter-binance-BNB-coin-token-blockchain-business-CZ-etoro-368x276.jpg")
+    no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.fourthSectionBoxLatestRight6 {
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/adult-content-cryptocurrency.jpg")
+    no-repeat;
+  background-size: cover;
+  background-position: 50%;
+  width: 30%;
+}
+
+.fourthSectionBoxImg7 {
+  margin-top: 15px;
+  height: 276px;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/06/Orion-Doom-Bethesda-hed-368x276.jpg")
+    no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.fourthSectionBoxLatestRight7 {
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/04/firefox-stop-cryptojacking-bitcoin-mining-cryptocurrency-monero-blockchain-mozilla.jpg")
+    no-repeat;
+  background-size: cover;
+  background-position: 50%;
+  width: 30%;
+}
+
+.fourthSectionBoxImg8 {
+  margin-top: 15px;
+  height: 276px;
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2017/09/multiplayer-gaming-368x276.jpg")
+    no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.fourthSectionBoxLatestRight8 {
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2018/08/Ninja-Twitter.jpg")
+    no-repeat;
+  background-size: cover;
+  background-position: 50%;
+  width: 30%;
+}
+
+/* FIFTH SECTION */
+
+#fifthSection {
+  display: grid;
+  width: 80%;
+  margin: auto;
+  margin-top: 80px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 40px 320px 300px;
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
+}
+
+figure {
+  padding: 0;
+  margin: 0;
+}
+
+.fifthSectionBox figcaption {
+  height: 50%;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.fifthSectionBox img {
+  width: 100%;
+  height: 220px;
+}
+
+.fifthSectionAuthor {
+  color: black;
+}
+
+.fifthSectionAuthor span {
+  color: gray;
+  font-size: 13px;
+  font-weight: 400;
+  text-decoration: line-through;
+}
+
+.fifthSectionAuthor b {
+  color: black;
+  font-size: 16px;
+  font-weight: 400;
+  text-decoration: none;
+}
+
+@media screen and (max-width: 1024px) {
+  #fifthSection {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 40px 320px 320px 320px;
+    margin-bottom: -50px;
+  }
+
+  #latestDeals {
+    grid-column: 1 / 4;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  #fifthSection {
```

Comment:
`HINT`

You seem to be using only two breakpoints for the media query which is affecting the display of this section for smaller screens. See below:

| Current | Preferred |
| :-----: | :-----: |
| ![tnw current](https://user-images.githubusercontent.com/36057474/62747638-47b80080-ba4d-11e9-96b5-5efafb919936.png) | ![tnw preffered](https://user-images.githubusercontent.com/36057474/62747646-4d154b00-ba4d-11e9-86df-c51d342cb631.png) |